### DISCUSSION TITLE
Building with separate toolbar

### DISCUSSION THREAD

**Discussion by jelabarre59** [pdfarranger/pdfarranger#1107] [General]
Is it possible to build PDFarranger with a separate toolbar, rather than having all the tool icons shoved into the titlebar?  It looks really bad under Cinnamon Desktop having the titlebar filled with icons that don't belong there.

**Comment by dreua** [comment]
I'm not aware of an easy way to do that unfortunately. This should be a problem not only with PDF Arranger but all apps designed for gnome, right? (e.g. Meld, ZIM, Helvum...) Btw, where are the standard apps like file manager, PDF and image viewer etc come from in cinnamon? If it is based on the Gnome apps then I wonder what they did to mitigate this issue.

**Reply by jelabarre59** [reply]
A lot of them have been forked from the Gnome versions (Caja, Xed, Xreader, etc)

Of course, I think Cinnamon should just port over to CTK anyway.
